scripts/config.pl

```diff
@@ -104,6 +104,7 @@
 MBEDTLS_PSA_CRYPTO_SPM
 MBEDTLS_PSA_INJECT_ENTROPY
 MBEDTLS_ECP_RESTARTABLE
+MBEDTLS_TLS_DEFAULT_ALLOW_SHA1_IN_CERTIFICATES
```

Comment:
This should probably change along with the disabling of the option by default, in that same commit.

Reply:
This option has been disabled by default since its inception. Before, it was in a section that `config.pl` didn't touch, so it didn't need to be mentioned in `config.pl`. Now that `config.pl full` processes options in the whole file, this one needs to be excluded explicitly to avoid enabling it (which would be bad because it's a security risk).